api/envoy/config/filter/network/dubbo_proxy/v2alpha1/route.proto

```diff
@@ -0,0 +1,104 @@
+syntax = "proto3";
+
+package envoy.config.filter.network.dubbo_proxy.v2alpha1;
+option java_package = "io.envoyproxy.envoy.config.filter.network.dubbo_proxy.v2alpha1";
+option java_multiple_files = true;
```

Comment:
can you merge master? I think this line is no longer needed (will fail master CI)